src/utils/parse-comment/index.js

```diff
@@ -22,16 +23,58 @@ const validContributionTypes = [
     'tool',
     'translation',
     'tutorial',
-    'userTesting',
+    'usertesting',
     'video',
 ]
 
+// Types that are valid multi words, that we need to re map back to there camelCase parts
+const validMultiContributionTypesMapping = {
+    eventorganizing: 'eventOrganizing',
+    fundingfinding: 'fundingFinding',
+    usertesting: 'userTesting',
+}
+
+// Additional terms to match to types (plurals, aliases etc)
```

Comment:
I investigated using compromise for plurals and it fell quite shot (got 15 of our needed 21) also missed some plural forms.